index.ts

```diff
@@ -62,5 +62,14 @@ export {renewSession, onSessionRenewed, logout} from './src/sessions';
 
 export {showAppRating} from './src/app-rating';
 
-export {bottomSheet, bottomSheetSingleSelector} from './src/bottom-sheet';
-export type {SheetListItem} from './src/bottom-sheet';
+export {
+    bottomSheet,
+    bottomSheetSingleSelector,
+    bottomSheetActionSelector,
+    bottomSheetInfo,
```

Comment:
Perhaps the bridge responsibility should be to provide the bottomSheet feature.

The bottomSheetSingleSelector, bottomSheetActionSelector, bottomSheetInfo or any other future variant should be implemented in the application.

Reply:
I think these wrapper functions are quite convenient for the final apps (webapp, solar) to use, given the SheetUI model is quite complex. I think this is the best place to have these functions. In mistica we'll have the web implementations of these components, not these helper functions.